### ISSUE TITLE
Exceptions that don't occur on a route produce error.

### ISSUE THREAD

**Issue by DFurnes** [DoSomething/northstar#454]
We just [added some code](https://github.com/DoSomething/northstar/pull/452) to always show JSON error messages on any routes that use the `'api'` middleware group. This seems to cause an error if an exception occurs when not on a specific route:

```
[2016-10-14 14:46:37] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function middleware() on null' in /var/www/northstar/releases/20161014140847/app/Exceptions/Handler.php
```

**Comment by sergii-tkachenko** [comment]
I'm getting HTTP 500 on Northstar-Thor:
```json
{
    "error": {
        "code": 500,
        "message": "Looks like something went wrong. We've noted the problem and will try to get it fixed!"
    }
}
```

Is this it?

**Comment by DFurnes** [comment]
That's an unrelated issue – see #api channel. But it was what caused me to notice this one. 😅

**Comment by DFurnes** [comment]
D'oh, this should be fixed now by #455! 🔨
